doc/source/dev/contributor/runtests.rst

```diff
@@ -2,9 +2,9 @@
 
 .. _runtests:
 
-===========================
-Running SciPy Tests Locally
-===========================
+================================================
+Running SciPy Tests Locally with ``runtests.py``
```

Comment:
What about we remove this page?

Reply:
Yup! Just didn't want to do it live and risk missing something. Will finish this up later today!